### ISSUE TITLE
Moving `fluxcat.py` to a Dedicated Repository

### ISSUE THREAD

**Issue by ssiegelx** [chime-experiment/ch_util#89]
I propose migrating `fluxcat.py` from `ch_util` to a new, lightweight repository within the `radiocosmology` organization. There is nothing CHIME-specific about `fluxcat` and this change would make it more accessible and useful to other experiments.

## Problem: Catalog File Dependency
`fluxcat` relies on catalog files originally generated for its use but currently stored in `ch_ephem`. We need to decide on the best location for these files.

### Proposed Solutions
1. **Centralize Catalog Files in `fluxcat`:**
   - Move all catalog files to the new `fluxcat` repository.
   - Update `ch_ephem` to import `fluxcat` and access the source coordinates it requires from the `FluxCatalog` object.

2. **Separate Reduced Catalog for `ch_ephem`:**
   - Store the full catalog files with flux measurements from other telescopes in the `fluxcat` repository.
   - Create a pared-down set of catalogs containing only the source coordinates for use in `ch_ephem`.
   - *Risk:* This approach introduces the potential for discrepancies in source coordinates between the two repositories over time.

### Request for Feedback
- Does centralizing the full catalog files in the `fluxcat` repository and updating `ch_ephem` to use them (i.e., proposal 1) seem like the best approach?
- Are there alternative strategies for managing catalog files that would avoid potential data divergence?
- Any there other considerations I might have overlooked?

I talked with Liam a bit about this at the recent hack-a-thon.  I'm also tagging Don since he was heavily involved in moving `ephemeris.py` out of `ch_util` and creating `ch_ephem`.

**Comment by ketiltrout** [comment]
Yeah, the interaction between fluxcat and what I pulled out of here for ch_ephem always felt a little unsatisfying...

I agree there's nothing CHIME-specific about the fluxcat code.  Are the catalogues CHIME-specific?  Or are they also generic enough to be in radiocosmology, too?

The dependencies of fluxcat per se seem fairly light.  If fluxcat is separated, I think making it, in turn, a dependency for ch_ephem would be fine (which I separated from ch_util to remove needing some of the heavy ch_util dependencies in projects only wanting the ephemeris).

What about `hfbcat`?  Would it go with fluxcat, or stay in ch_util?

**Comment by ljgray** [comment]
I think I'm happy with option 1 and centralize the catalogs in `fluxcat`. Since the goal is for these to be usable by other experiments, I think the new repo should be in `radiocosmology` and the catalogs updated as mentioned

**Comment by ljgray** [comment]
I'm going to have a crack at this, with the following steps:
- [ ] Make a new `fluxcat` repo in `radiocosmology`
- [ ] Move `fluxcat.py` and all the catalog files in `ch_ephem` to `fluxcat`
- [ ] Update `ch_ephem` to load files from `fluxcat`, keeping the current `ch_ephem.catalog` api and adding a deprecation warning

**Comment by ssiegelx** [comment]
That sounds great, thanks Liam!

**Comment by ljgray** [comment]
This is done. `ch_util.fluxcat` is removed in the `refactor_dev` branch.
